### DISCUSSION TITLE
Huge roundrect particle picks

### DISCUSSION THREAD

**Discussion by asarnow** [nextpyp/.github#109] [Q&A] [answered]
This is AreTomo align and reconstruct without square padding, particle radius is set to 150 Å.

<img width="541" alt="image" src="https://github.com/nextpyp/.github/assets/4052606/8fd57ca8-66e0-4a7f-bea2-1acb7514bf7b">

**Comment by abartesaghi** [comment, marked as answer]
This was caused by a bug that prevented the AreTomo reconstruction from being recalculated when changes were made to the "Tomogram reconstruction" tab. The fix will be part of the next release. In the meantime, if you force recalculation of your tomograms (for example, by selecting IMOD as the "Method for tomogram reconstruction") and re-run the Pre-processing block, this issue should go away.

**Reply by asarnow** [reply]
Thanks! That does fix the picking issue, but the dimensions of the imod tomograms in this case are rotated, so the content is cropped.

AreTomo -> AreTomo

<img width="478" alt="image" src="https://github.com/nextpyp/.github/assets/4052606/7f5a1c16-4a81-4774-b8d2-e334ef15ce07">

AreTomo -> IMOD

<img width="485" alt="image" src="https://github.com/nextpyp/.github/assets/4052606/9cbf3151-fa44-41ed-a211-5f524cc7d299">

**Reply by abartesaghi** [reply]
We can't seem to be able to reproduce this one, are you using version 0.6.5? Also, in some browsers the image from the previous run may be cached, does refreshing the page make any difference? When you say "AreTomo -> IMOD", are you referring to just changing the option "Method for tomogram reconstruction" from one method to the other?

**Reply by asarnow** [reply]
That's correct, I meant that the alignments were in AreTomo and the reconstructions were AreTomo or IMOD. I only recalculated tomograms by changing the option in a completed preprocessing block and rerunning it, not by creating a new processing block. I'll try that too now.

At the time I took the screenshots, the job was still running, and tomograms with an updated calculation date done by IMOD were horizontal while the old ones done by AreTomo were vertical. (Again, this is just the dimension, not the alignments, you can see that the content itself remains well-aligned and vertical in both cases).

Now that the job is complete, with all TS processed successfully, they are all horizontal. However, the gallery previews are a mixture of vertical and horizontal, I supposed they were not all recreated somehow. The table view previews are all updated with the new, incorrectly horizontal, reconstructions.

And yes 0.6.5 with the hotfix to display 0.6.5.

Additional notes:
 - Frame alignment by MC3
 - AreTomo reconstructions used SART
 - AreTomo NOT estimating tilt offsets
 - Tilt images NOT reshaped to square
 - No denoising applied

**Reply by asarnow** [reply]
I did a new preprocessing block with AreTomo alignments and IMOD reconstructions, with reshaping to square, and the picks are oval for that block as well.

**Reply by asarnow** [reply]
Last update - I finished rerunning the 1st block, still with IMOD reconstructions but now reshaping to squares. The picks are circular, but the tomograms are still cut off by the shape of the tilt series (they did not become squares).
